<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLViewer</class>
 <widget class="QMainWindow" name="PCLViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidgetDetect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>971</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_Model">
     <attribute name="title">
      <string>Model</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_BrowseModel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QVTKWidget" name="qvtkWidgetModel" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>1021</width>
        <height>611</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Calibration</string>
     </attribute>
     <widget class="QLabel" name="labelCameraMatrixRobot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transform Matrix from Camera to Robot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_BrowseTransformMatrix">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMatrixText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>541</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>T = 






</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_SyntheticDetection">
     <attribute name="title">
      <string>Synthetic Detection</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_StartSimulation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Simulation</string>
      </property>
     </widget>
     <widget class="QVTKWidget" name="qvtkWidgetSimulation" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>1031</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SimulationDetectObjects">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Detect Objects</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxVisualize">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualize</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_ConnectToNachiSocket">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect to Nachi Socket</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_RealtimeDetection">
     <attribute name="title">
      <string>Realtime Detection</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidgetRealtimeDetection">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>951</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_2D">
       <attribute name="title">
        <string>2D</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_StartRealsenseCamera">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>10</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Realsense Camera</string>
        </property>
       </widget>
       <widget class="QLabel" name="RealsenseImageLabel">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>80</y>
          <width>640</width>
          <height>480</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3D">
       <attribute name="title">
        <string>3D</string>
       </attribute>
       <widget class="QVTKWidget" name="qvtkWidgetCamera3D" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>900</width>
          <height>420</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Points">
       <attribute name="title">
        <string>Points</string>
       </attribute>
       <widget class="QVTKWidget" name="qvtkWidgetRemoveBackground" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>900</width>
          <height>580</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_RemoveBackground">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Remove background</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Objects">
       <attribute name="title">
        <string>Objects</string>
       </attribute>
       <widget class="QVTKWidget" name="qvtkWidgetObjects" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>900</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_DetectObjects">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Detect Objects</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Pick1stObject">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pick 1st Object</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_StartSequence">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Sequence</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_MoveToCamera0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>0</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Move to Camera 0</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
